test/resources/selinux.py

```diff
@@ -0,0 +1,119 @@
+from robot.libraries.BuiltIn import BuiltIn
+from libostree import remote_sudo_rc, remote_sudo
+from typing import List
+
+ACCESS_CHECK_MAP = {
+    "/var/lib/microshift/version": ["cat"],
+    "/etc/microshift/config.yaml.default": ["cat"],
+}
+
+CONTEXT_CHECK_MAP = {
+    "system_u:object_r:container_var_lib_t:s0": [
+        "/var/lib/microshift",
+        "/var/lib/microshift-backups",
+    ],
+    "system_u:object_r:kubelet_exec_t:s0": [
+        "/usr/bin/microshift",
+        "/usr/bin/microshift-etcd",
+    ],
+    "system_u:object_r:kubernetes_file_t:s0": [
+        "/etc/microshift",
+    ],
+}
+
+EXPECTED_FCONTEXT_LIST = [
```

Comment:
Testing the effect, that the files and directories have the right labels, seems like the most important bit. If managing this list requires a lot of maintenance over time, I'd consider dropping the test step that uses it and focus on the labels on files unique to MicroShift.

Reply:
That's a good point, I can add more access checks on our files and directories. 

In regard to this list, it should only ever change if we alter the SELinux policy or upstream container linux package changes something in these contexts, those events should be rare. Regardless, I think this is a good canary check if anything changes these contexts, so we can decide what that means for MicroShift.

Reply:
@eggfoobar , can you add this to the comments in the file?

Reply:
Sure thing. Added the comment